binrz/rz-test/rz_test.h

```diff
@@ -60,16 +60,16 @@ typedef struct rz_test_cmd_test_t {
 } RzCmdTest;
 
 #define RZ_CMD_TEST_FOREACH_RECORD_NOP(name, field)
-#define RZ_CMD_TEST_FOREACH_RECORD(macro_str, macro_bool, macro_int) \
-	macro_str ("NAME", name) \
-	macro_str ("FILE", file) \
-	macro_str ("ARGS", args) \
-	macro_int ("TIMEOUT", timeout) \
-	macro_str ("SOURCE", source) \
-	macro_str ("CMDS", cmds) \
-	macro_str ("EXPECT", expect) \
-	macro_str ("EXPECT_ERR", expect_err) \
-	macro_bool ("BROKEN", broken)
+#define RZ_CMD_TEST_FOREACH_RECORD(macro_str, macro_bool, macro_int)                                \
+	macro_str("NAME", name)                                                                     \
+		macro_str("FILE", file)                                                             \
+			macro_str("ARGS", args)                                                     \
+				macro_int("TIMEOUT", timeout)                                       \
+					macro_str("SOURCE", source)                                 \
+						macro_str("CMDS", cmds)                             \
+							macro_str("EXPECT", expect)                 \
+								macro_str("EXPECT_ERR", expect_err) \
+									macro_bool("BROKEN", broken)
```

Comment:
This looks bad

Reply:
I know and reported to Clang developers long time ago - no reaction so far: https://bugs.llvm.org/show_bug.cgi?id=46240

Reply:
I guess the problem here is that there is no `;` so it does not know that those `macro_str` expand to a `if` block. Maybe this could be fixed by using `;` after the `macro_str` and adjusting the `DO_KEY_*` macros so that they are wrapped in `do { ... } while (0)` (due to the `continue` in those blocks, maybe a `goto continue_block` can be used instead of a direct `continue`.

Reply:
We can fence this off with `// clang-format off`

Reply:
Try `StatementMacros` first (from https://clang.llvm.org/docs/ClangFormatStyleOptions.html ). I think this is our case.